### PR TITLE
[RFC] Update documentation according to the latest changes

### PR DESCRIPTION
**Pull request by kostyanf14**
Signed-off-by: Kostiantyn Kostiuk <konstantin@daynix.com>

### CODE REVIEW

File: IMAGE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# AutoHCK image creation
+
+AutoHCK is a tool for automating HCK/HLK testing, doing all the boilerplate steps in the process leaving you with simply choosing which driver you want to test on what os.
```
Comment (YanVugenfirer) [inline IMAGE.md:3]: "os" -> "OS"

File: IMAGE.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+   - **path** - contains the path relative to `iso_path` option in config file to corresponding ISO image;
+   - **windows_image_names** - contains image name from `install.wim` file in ISO. To get a list of available images use the next commands:
+      * `dism /get-wiminfo /wimfile:D:\source\install.wim` on Windows;
+      * `wiminfo /mnt/source/install.wim` on Linux (wimlib is required);
+   - **product_key** - contains a valid key for the corresponding Windows image. Generic keys from MSDN can be used [KMS Client Setup Keys](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/jj612867(v=ws.11));
+
+2. `lib/engines/hckinstall/hckinstall.json` - contains the specific configuration for install engine. The following fields should be configured:
+   - **hck_setup_scripts_path** - path to HLK-Setup-Scripts repository;
+   - **answer_files** - list of the unattended answer file for automatic Windows installation. These files are part of HLK-Setup-Scripts repository;
+   - **studio_install_timeout** - timeout before the installation of the Studio VM is considered a failure;
```
Comment (YanVugenfirer) [inline IMAGE.md:25]: I would mark timeouts as the advanced configuration that should be changed only if the user understands why.

File: IMAGE.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+   - **client_install_timeout** - timeout before the installation of the Client VMs is considered a failure;
+
+3. `config.json` - contains the general AutoHCK configuration. The following fields should be configured:
+   - **iso_path** - absolute path to the directory where ISO stored
+
+4. `lib/engines/hckinstall/kit.json` - contains information about each HCK/HLK kit. The following fields should be configured:
+   - **kit** - The kit name should be the same as in the `platforms.json` file;
+   - **path** - contains the path for download the kit installer;
+
```
Comment (YanVugenfirer) [inline IMAGE.md:34]: I suggest having the above section as detailed as possible so a user can follow the steps without thinking. What I am missing is ISO copy/storage.

**Comment by YanVugenfirer** [comment]
Also, maybe call the file AutoImageInstallation.md or ImageInstallation.md

File: IMAGE.md (hunk 3 above)
Comment (Basils) [reply IMAGE.md:34]: I think the documentation for kit.json is related to patches that were not merged/sent yet.
kit.json file does not exist and i can't find where the current implementation uses the file.

**Review by Basils — CHANGES_REQUESTED** [review]
some line endings have ; (semicolon) instead of  . (fullstop)

**Comment by kostyanf14** [comment]
Updated files according to your comments. Please review again.

**Review by YanVugenfirer — APPROVED** [review]

**Pull request merged**